themes/classic/_dev/css/partials/_helpers.scss

```diff
@@ -0,0 +1,49 @@
+/*** HELPERS ***/
+
+._relative{
+  position: relative;
+}
+
+._dpt{
+  display: table;
+}
```

Comment:
1. naming, 2. I'm sure that bootstrap has fwe of this helpers already and it will be better to use them?

It is golden rule that we shouldn't name things like this, no one will understand dpt or mlr, remember that most of people will use Classic as their "starter theme" :)